nibabies/workflows/bold/registration.py

```diff
@@ -472,8 +472,7 @@ def init_bbreg_wf(use_bbr, bold2t1w_dof, bold2t1w_init, omp_nthreads, name='bbre
     if bold2t1w_init == "header":
         bbregister.inputs.init = "header"
 
-    transforms = pe.Node(niu.Merge(1 if bold2t1w_init == "header" else 2),
-                         run_without_submitting=True, name='transforms')
+    transforms = pe.Node(niu.Merge(2), run_without_submitting=True, name='transforms')
     lta_ras2ras = pe.MapNode(LTAConvert(out_lta=True), iterfield=['in_lta'],
                              name='lta_ras2ras', mem_gb=2)
     select_transform = pe.Node(niu.Select(), run_without_submitting=True, name='select_transform')
```

Comment:
```suggestion
    # Default to taking first available transform
    select_transform = pe.Node(niu.Select(index=0), run_without_submitting=True, name='select_transform')
```

Reply:
i don't think we can do this, since in some cases it won't always be the first

https://github.com/nipreps/nibabies/blob/0ba3cdc5342f339f7f5e6dbecf46f2b5200bcf34/nibabies/workflows/bold/registration.py#L544

Reply:
Right, in which case it will be overridden by the connection.

Reply:
right...i seem to have forgotten how nipype works.